src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.TalonFXControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+/** Add your docs here. */
+public class DriveTrain extends SubsystemBase {
+  public TalonFX leftLeader = new TalonFX(Constants.driveLeftLeaderFalconID);
+  public TalonFX leftFollower = new TalonFX(Constants.driveLeftFollowerFlaconID);
+  public TalonFX rightLeader = new TalonFX(Constants.driveRightLeaderFalconID);
+  public TalonFX rightFollower = new TalonFX(Constants.driveRightFollowerFalconID);
+
+  // Put methods for controlling this subsystem
+  // here. Call these from Commands.
+
+  public DriveTrain() {
+    leftLeader.configFactoryDefault();
+    leftFollower.configFactoryDefault();
+    rightLeader.configFactoryDefault();
+    rightFollower.configFactoryDefault();
+
+    leftFollower.setInverted(false);
+    rightFollower.setInverted(true);
+
+    leftFollower.follow(leftLeader);
+    rightFollower.follow(rightLeader);
+
+  }
+
+  public void tankDrive(double leftSpeed, double rightSpeed) {
+
+    leftLeader.set(TalonFXControlMode.PercentOutput, leftSpeed);
+    rightLeader.set(TalonFXControlMode.PercentOutput, rightSpeed);    
+  }
+
+  public void arcadeDrive(double speed, double turn) {
+    turn = MathUtil.clamp(turn, -1* Constants.kTurnClamp, Constants.kTurnClamp);
```

Comment:
`kTurnClamp` was initialized to `0`, which means that we're clamping our turn value between `-0` and `0`- that's probably not what we want, right?

Reply:
I didn't assign any true values for any constants. I used fillers for now, but yes, it will be changed.

Reply:
okay, that makes sense :) can you add a `TODO` comment to for `kTurnClamp` in the constants class to tune this?

Reply:
Yep!